shared/gh/js/views/gh.admin-batch-edit-date.js

```diff
@@ -86,45 +90,56 @@ define(['lodash', 'moment', 'gh.api.util', 'gh.api.config'], function(_, moment,
     /**
      * Add another day to the terms based on the selection of weeks
      *
+     * @param {Number}    [weekNumber]    The optional number of the week to add the events to
      * @private
      */
     var addAnotherDay = function() {
```

Comment:
To clarify, I see your point but since no unexpected input will pass through those it's probably not worth the effort at the moment.